.travis.yml

```diff
@@ -40,15 +41,36 @@ matrix:
       addons: {"apt":{"sources":["hvr-ghc"],"packages":["ghc-8.2.2","cabal-install-2.4"]}}
     - compiler: ghc-8.0.2
       addons: {"apt":{"sources":["hvr-ghc"],"packages":["ghc-8.0.2","cabal-install-2.4"]}}
+  allow_failures:
+    - compiler: ghc-8.8.1
```

Comment:
```
% cabal new-build -w ghc-8.8.1 all
Resolving dependencies...
cabal: Could not resolve dependencies:
[__0] trying: generic-optics-0.1 (user goal)
[__1] trying: profunctors-5.4 (dependency of generic-optics)
[__2] trying: tagged-0.8.6 (dependency of profunctors)
[__3] trying: template-haskell-2.15.0.0/installed-2.1... (dependency of
tagged)
[__4] trying: base-4.13.0.0/installed-4.1... (dependency of generic-optics)
[__5] trying: metametapost-0.1 (user goal)
[__6] next goal: generics-sop (dependency of metametapost)
[__6] rejecting: generics-sop-0.5.0.0, generics-sop-0.4.0.1,
generics-sop-0.3.2.0 (conflict: template-haskell==2.15.0.0/installed-2.1...,
generics-sop => template-haskell>=2.8 && <2.15)
...
[__6] fail (backjumping, conflict set: base, generics-sop, metametapost,
template-haskell)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: template-haskell, tagged, base,
profunctors, generics-sop, generic-optics, metametapost
```

I.e. without relaxing bounds (and possibly `head.hackage`), full optics suite cannot be built.

Reply:
that to say: if one tests with extensive `--allow-newer` and/or head.hackage patches, it's testing against unreleased or unsupported configurations. And in fact, it does fail randomly.

Reply:
Okay, I guess we might indeed end up with false failures; it would just be a shame if we accidentally broke 8.8 support ourselves. But I guess that is relatively unlikely, at least in the short term, and we can revisit this when the ecosystem has caught up a bit.